lib/actions/action.js

```diff
@@ -31,7 +31,7 @@ class Action extends EventAware {
   logBeforeValidateUsage (context, settings) {
     const usageLog = {
       log_type: logger.logTypes.ACTION_BEFORE_VALIDATE_EXECUTE,
-      eventId: context.eventId,
+      event_id: context.eventId,
```

Comment:
We've been using camelCase throughout the codebase except for these properties which is snake case. If we want to be consistent, should all the properties in this JSON be changed to camelCase instead?

Reply:
what are your thoughts? i am okay with either, i just notice this one is out of place. Our log types are in `snake_case` too, maybe we should change those too

Reply:
LogTypes are constants in the traditional sense are all caps and because of that snake case. I'd leave those alone.

I prefer to be consistent with camelcase for the properties since that has been widely the convention.

Reply:
sure, i'll change them